package.json

```diff
@@ -35,12 +35,14 @@
         "chart.js": "^2.7.3",
         "clipboard-copy": "^2.0.0",
         "prop-types": "^15.6.2",
-        "react": "^16.8.6",
         "react-async-script-loader": "^0.3.0",
-        "react-dom": "^16.8.6",
-        "styled-components": "^4.3.2",
         "xlsx": "^0.14.4"
     },
+    "peerDependencies": {
+        "react": "^16.8.6",
+        "react-dom": "^16.8.6",
```

Comment:
We should not restrict a specific version of react, instead, we should let the user use whatever version greater than 16.8.0 since this is the version where react introduce the hooks.
```
"react": ">=16.8.0",
"react-dom": ">=16.8.0"
```